src/screens/Onboarding/StepFinished.tsx

```diff
@@ -14,6 +15,12 @@ import {useAgent} from '#/state/session'
 import {useOnboardingDispatch} from '#/state/shell'
 import {uploadBlob} from 'lib/api'
 import {useRequestNotificationsPermission} from 'lib/notifications/notifications'
+import {makeStarterPackLink} from 'lib/routes/links'
+import {
+  useCurrentStarterPack,
+  useSetCurrentStarterPack,
+} from 'state/preferences/starter-pack'
```

Comment:
Is this being saved per-device (prefs API) rather than per-account? Is it being synced between tabs?

Reply:
Totally wiped out storing anything here, just using a context higher up the tree.